packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import { cx } from 'emotion';
+import { useTheme } from '../../themes';
+import { IconType } from '../Icon/types';
+import { getTabsStyle } from './styles';
+
+export interface TabProps {
+  label: string;
+  active?: boolean;
+  icon?: IconType;
+  onChangeTab: () => void;
+}
+
+export const Tab: FC<TabProps> = ({ label, active, icon, onChangeTab }) => {
+  const theme = useTheme();
+  const tabsStyles = getTabsStyle(theme);
+
+  return (
+    <li className={cx(tabsStyles.tabItem, active && tabsStyles.activeStyle)} onClick={onChangeTab}>
+      {icon}
```

Comment:
This is not going to work with IconType -> You need to render `{ icon && <Icon name={icon} /> }`